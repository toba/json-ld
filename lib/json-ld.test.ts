import ld from './json-ld';
