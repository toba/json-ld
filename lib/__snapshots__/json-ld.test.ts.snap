// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders breadcrumb correctly 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Breadcrumb",
  "item": Object {
    "id": "http://example.com/page/",
    "name": "First Title",
  },
  "position": 1,
}
`;

exports[`renders breadcrumb correctly 2`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Breadcrumb",
  "item": Object {
    "id": "http://example.com/page/",
    "name": "Second Title",
  },
  "position": 2,
}
`;

exports[`renders discover action correctly 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "DiscoverAction",
  "target": "http://example.com/page/",
}
`;

exports[`renders image correctly 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "ImageObject",
  "height": 600,
  "url": "http://example.com/photo.jpg",
  "width": 800,
}
`;

exports[`renders organization correctly 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Organization",
  "logo": Object {
    "@context": "http://schema.org",
    "@type": "ImageObject",
    "height": 600,
    "url": "http://example.com/photo.jpg",
    "width": 800,
  },
  "name": "Org With Logo",
}
`;

exports[`renders organization correctly 2`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Organization",
  "name": "Org Without Logo",
}
`;

exports[`renders place correctly 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Place",
  "hasMap": "http://example.com/page/",
}
`;

exports[`renders webPage correctly 1`] = `
Object {
  "@context": "http://schema.org",
  "@id": "http://example.com/page/",
  "@type": "WebPage",
}
`;
